lstchain/io/io.py

```diff
@@ -443,6 +445,239 @@ def read_array_info(filename):
     return array_info
 
 
+def read_optic(filename, telescope_name):
+    """
+    Read a specific telescope optics from a DL1 file
+
+    Parameters
+    ----------
+    filename: str
+    telescope_name: str
+
+    Returns
+    -------
+    `ctapipe.instrument.optics.OpticsDescription`
+    """
+    from astropy.units import Quantity
+    telescope_optics_path = "/configuration/instrument/telescope/optics"
+    telescope_optic_table = Table.read(filename, path=telescope_optics_path)
+    row = telescope_optic_table[np.where(telescope_name == telescope_optic_table['name'])[0][0]]
+    optics_description = OpticsDescription(
+        name=row['name'],
+        num_mirrors=row['num_mirrors'],
+        equivalent_focal_length=row['equivalent_focal_length'] * telescope_optic_table['equivalent_focal_length'].unit,
+        mirror_area=row['mirror_area'] * telescope_optic_table['mirror_area'].unit,
+        num_mirror_tiles=Quantity(row['num_mirror_tiles']),
+    )
+    return optics_description
+
+
+def read_optics(filename):
+    """
+    Read all telescope optics from a DL1 file
+
+    Parameters
+    ----------
+    filename: str
+
+    Returns
+    -------
+    dictionnary of ctapipe.instrument.optics.OpticsDescription by telescope names
+    """
+    telescope_optics_path = "/configuration/instrument/telescope/optics"
+    telescope_optics_table = Table.read(filename, path=telescope_optics_path)
+    optics_dict = {}
+    for telescope_name in telescope_optics_table['name']:
+        optics_dict[telescope_name] = read_optic(filename, telescope_name)
+    return optics_dict
+
+
+def read_camera_geometry(filename, camera_name):
+    """
+    Read a specific camera geometry from a DL1 file
+
+    Parameters
+    ----------
+    filename: str
+    camera_name: str
+
+    Returns
+    -------
+    `ctapipe.instrument.camera.geometry.CameraGeometry`
+    """
+    camera_geometry_path = f"/configuration/instrument/telescope/camera/geometry_{camera_name}"
+    camera_geometry = CameraGeometry.from_table(Table.read(filename, camera_geometry_path))
+    return camera_geometry
+
+
+def read_camera_geometries(filename):
```

Comment:
can we use something more intuitive like `read_camera_geometry_table` whenever it is more than one?
Same for the other functions

Reply:
Ditto